opm/parser/eclipse/EclipseState/Schedule/Well/WList.hpp

```diff
@@ -28,29 +28,33 @@ namespace Opm {
 
 class WList {
 public:
-    using storage = std::unordered_set<std::string>;
+    using storage = std::vector<std::string>;
```

Comment:
I think `std::map<std::string, std::size_t>` would be a simpler solution than `std::vector`.